web_server/authors/views.py

```diff
@@ -225,10 +211,11 @@ def create_new_post(request):
         new_post.title = post['title']
         # new_post.source      = post['source']       #: "http://lastplaceigotthisfrom.com/posts/yyyyy"
         # new_post.origin      = post['origin']       #: "http://whereitcamefrom.com/posts/zzzzz"
-        new_post.description = post['description']  # : "This post discusses stuff -- brief",
+        # : "This post discusses stuff -- brief",
+        new_post.description = post['description']
         new_post.contentType = post['contentType']  # : "text/plain",
-        new_post.content     = post['content']      #: "stuffs",
-        new_post.author      = request.user         # the authenticated user
+        new_post.content = post['content']      #: "stuffs",
+        new_post.author = request.user         # the authenticated user
```

Comment:
This is an example of an area of the code you keep changing that is reducing readability. It's not a huge issue so I'm going to approve and merge, but please prevent your IDE from automatically changing the code others keep formatting to keep readable.